doc/rofi.1.markdown

```diff
@@ -665,21 +657,21 @@ The theme format below describes the old (pre version 1.4) theme format. Please
 manual.
 
 The theme setup allows you to specify colors per state, similar to **i3**
-Currently 3 states exists:
+Currently 3 states exist:
 
-  * **normal** Normal row.
-  * **urgent** Highlighted row (urgent)
-  * **active** Highlighted row (active)
+  * **normal**: normal row
+  * **urgent**: highlighted row (urgent)
+  * **active**: highlighted row (active)
 
-For each state the following 5 colors must be set:
+For each state, the following 5 colors must be set:
 
-  * **bg** Background color row
-  * **fg** Text color
-  * **bgalt** Background color alternating row
-  * **hlfg** Foreground color selected row
-  * **hlbg** Background color selected row
+  * **bg**: background color row
+  * **fg**: text color
+  * **bgalt**: background color alternating row
+  * **hlfg**: foreground color selected row
+  * **hlbg**: background color selected row
 
-The window background and border color should be specified separate. The key `color-window` contains
+The window background and border color should be specified separately. The key `color-window` contains
 a pair `background,border,separator`.
```

Comment:
changing pair -> tuple
I forgot that a tuple doesn't only consist of two elements :man_shrugging: